components/typography/typography.scss

```diff
@@ -140,10 +140,6 @@
 	font-size: 0.8rem;
 	font-weight: 400;
 	line-height: 1.2rem;
-	@media (max-width: 615px) {
-		font-size: 0.8rem;
-		line-height: 1.2rem;
-	}
```

Comment:
This wasn't doing anything since it's setting the font size and line-height to the same values as they already were.